services/app-graphql/src/schema.graphql

```diff
@@ -4,36 +4,139 @@ schema {
 }
 
 type Query {
+    """
+    fetchCurrentUser returns user information for the currently logged in User
+
+    If no user is currently logged in, the http request will return a 403 error, no graphQL body will be returned.
+    """
     fetchCurrentUser: User!
+
+    """
+    fetchHealthPlanPackage returns a specific HealthPlanPackage by id
+
+    If a package with the given ID cannot be found, this query returns undefined
+    CMS users cannot fetch a DRAFT HealthPlanPackage
+
+    Errors:
+    - ForbiddenError:
+        - A state user from a different state called this.
+        - A CMSUser attempted to fetch a DRAFT HealthPlanPackage
+    """
     fetchHealthPlanPackage(
         input: FetchHealthPlanPackageInput!
     ): FetchHealthPlanPackagePayload!
+
+    """
+    indexHealthPlanPackages returns all of the HealthPlanPackages the current user can see.
+
+    StateUsers can find all the packages for their state
+    CMSUsers can find all the packages that do not have the DRAFT status
+    """
     indexHealthPlanPackages: IndexHealthPlanPackagesPayload!
 }
 
 type Mutation {
+    """
+    createHealthPlanPackage creates a new HealthPlanPackage.
+
+    The new HealthPlanPackage is created with a single revision and with
```

Comment:
I think someone new to the project would wonder what a "revision" is in this context.